src/frame/headers.rs

```diff
@@ -343,6 +342,17 @@ impl<T> From<Headers> for Frame<T> {
     }
 }
 
+impl fmt::Debug for Headers {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("Headers")
+            .field("stream_id", &self.stream_id)
+            .field("stream_dep", &self.stream_dep)
+            .field("flags", &self.flags)
+            // `fields` and `pseudo` purposefully not included
```

Comment:
Do you think this is an issue w/ trace level debugging? Removing headers will make debugging noticeably harder.